client/my-sites/plugins/controller.js

```diff
@@ -111,9 +111,11 @@ function renderPluginList( context, basePath ) {
 			: ''
 		);
 
-	analytics
-	.pageView
-	.record( context.pathname.replace( site.domain, ':site' ), analyticsPageTitle );
+	if ( site ) {
+		analytics
```

Comment:
I was thinking of leaving this for another PR. Main reasons:

* It's used the old way consistently throughout the controllers.
* `recordPageView` needs a service specified, while `analytics.pageView.record` records both a `ga` and `tracks` events.